ACE/ace/OS_NS_Thread.cpp

```diff
@@ -4472,7 +4472,7 @@ ACE_OS::thr_create (ACE_THR_FUNC func,
     }
 #   endif /* 0 */
 
-  if (tid == ERROR)
+  if (tid == TASK_ID_ERROR)
```

Comment:
According to VxWorks 5.5 the return value in older VxWorks versions is really ERROR, not TASK_ID_ERROR, see http://www.vxdev.com/docs/vx55man/vxworks/ref/taskLib.html#taskSpawn